fdb-record-layer-spatial/fdb-record-layer-spatial.gradle

```diff
@@ -26,24 +26,27 @@ if (!hasProperty('coreNotStrict')) {
 
 def coreProject = ":${ext.coreProjectName}"
 dependencies {
-    compile project(coreProject)
-    compile "org.locationtech.jts.io:jts-io-common:${jtsVersion}"
-    compile "com.geophile:geophile:${geophileVersion}"
-    compile "com.google.protobuf:protobuf-java:${protobufVersion}"
-    compile "org.slf4j:slf4j-api:${slf4jVersion}"
-    compile "org.slf4j:jul-to-slf4j:${slf4jVersion}"
+    implementation project(coreProject)
```

Comment:
For similar reasons to `fdb-record-layer-icu`, I think this should be an `api` dependency

Reply:
Fine with me.

Reply:
changed to api